### PR TITLE
EVG-6032 sorting and filtering, ensure type column is visible.

### PR DESCRIPTION
**Pull request by jimoleary**
The new list looks like:

image

### CODE REVIEW

File: public/static/app/perf-bb/PerfBBOutliersCtrl.js [hunk 1]
```diff
@@ -1,32 +1,42 @@
 mciModule.controller('PerfBBOutliersCtrl', function (
   $scope, $window, EvgUiGridUtil, EvgUtil, FORMAT, MDBQueryAdaptor, uiGridConstants,
-  STITCH_CONFIG, Stitch, Settings
+  STITCH_CONFIG, Stitch, Settings, $timeout, $compile, $log
 ) {
+  // Monkey Patch the toolbar as I can't change outliers.html.
```
Comment (jimoleary) [inline public/static/app/perf-bb/PerfBBOutliersCtrl.js]: This code will be deleted before the code is merged.

File: public/static/app/perf-bb/PerfBBOutliersCtrl.js [hunk 2]
```diff
@@ -229,21 +271,27 @@ mciModule.controller('PerfBBOutliersCtrl', function (
...
+        }).finally(() => $timeout(() => vm.isLoading = false));
+      // The finally and timeout clear loading on the next event loop iteration so that the Loading indicator
+      // disappears when the data load is complete. Otherwise there can be a gap.
     });
   }
 
+  vm.lowConfidenceChanged = () => {
+    $log.debug('Low Confidence changed : ' + vm.checkboxModel.lowConfidence);
+    vm.state.lowConfidence = vm.checkboxModel.lowConfidence;
+    vm.reload()
```
Comment (dbradf) [inline public/static/app/perf-bb/PerfBBOutliersCtrl.js]: [nit] semicolon?

File: public/static/app/perf-bb/PerfBBOutliersCtrl.js [hunk 3]
```diff
@@ -229,21 +271,27 @@ mciModule.controller('PerfBBOutliersCtrl', function (
...
+  };
+
   vm.modeChanged = () => {
     vm.state.mode = vm.mode.value;
-    vm.state.filtering = $scope.getDefaultFiltering(vm.state);
+    vm.reload()
```
Comment (dbradf) [inline public/static/app/perf-bb/PerfBBOutliersCtrl.js]: [nit] semicolon?

File: public/static/app/perf-bb/PerfBBOutliersCtrl.test.js [hunk 4]
```diff
@@ -42,25 +42,26 @@ describe('PerfBBOutliersCtrlTest', () => {
...
-    it('should filter by project, date, and type for suspicious', () => {
-      controller.mode.value = 'suspicious';
-      const filter = scope.getDefaultFiltering();
-
-      expect(filter.project).toEqual('=' + project);
-      const expectedDate = controller.state.lookBack.format(format.ISO_DATE);
-      expect(filter.create_time).toEqual('>' + expectedDate);
-      expect(filter.type).toEqual('=suspicious');
-    });
+    // it('should filter by project, date, and type for suspicious', () => {
```
Comment (dbradf) [inline public/static/app/perf-bb/PerfBBOutliersCtrl.test.js]: [nit] Can we delete this?

**Comment by jimoleary** [comment]
The test failure are in areas which were unchanged in this PR.

**Pull request merged**